ods_ci/tests/Tests/1000__model_serving/1009__model_serving_triton_on_kserve/1009__model_serving_triton_on_kserve.robot

```diff
@@ -0,0 +1,90 @@
+*** Settings ***
+Documentation     Suite of test cases for Triton in Kserve
+Library           OperatingSystem
+Library           ../../../../libs/Helpers.py
+Resource          ../../../Resources/Page/ODH/JupyterHub/HighAvailability.robot
+Resource          ../../../Resources/Page/ODH/ODHDashboard/ODHModelServing.resource
+Resource          ../../../Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Projects.resource
+Resource          ../../../Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/DataConnections.resource
+Resource          ../../../Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/ModelServer.resource
+Resource          ../../../Resources/Page/ODH/ODHDashboard/ODHDashboardSettingsRuntimes.resource
+Resource          ../../../Resources/Page/ODH/Monitoring/Monitoring.resource
+Resource          ../../../Resources/OCP.resource
+Resource          ../../../Resources/CLI/ModelServing/modelmesh.resource
+Suite Setup       Triton On Kserve Suite Setup
+Suite Teardown    Triton On Kserve Suite Teardown
+Test Tags         Kserve
+
+*** Variables ***
+${INFERENCE_REST_INPUT_ONNX}=    @tests/Resources/Files/triton/kserve-triton-onnx-rest-input.json
+${PRJ_TITLE}=    model-serving-triton-project
+${PRJ_DESCRIPTION}=    project used for model serving triton runtime tests
+${MODEL_CREATED}=    ${FALSE}
+${MODEL_NAME}=    densenet_onnx
```

Comment:
pay attention to the suite variable, you're conflicting with the same variables in a parallel PR https://github.com/red-hat-data-services/ods-ci/pull/1735/files#diff-d94a60fcd3111f1d0bae38f90b9327b1ddedbbd7f431fcd6b372098a94ee5bf8R23